Library/Formula/cdrtools.rb

```diff
@@ -45,6 +60,10 @@ def install
 
   test do
     system "#{bin}/cdrecord", "-version"
+    system "#{bin}/cdda2wav", "-version"
```

Comment:
I remember that 1 or 2 years ago there has been a (single) 3.01aXX preview release which failed to build cdda2wav. I do not remember, however, if the compilation silently skipped the cdda2wav target.or not. Given that cdrecord, mkisofs and cdda2wav are the main commands of cdrtools, and that the only one we can really test is mkisofs, I think it would be worth keeping a "-version" test for both cdrecord and cdda2wav. But if you prefer to have them removed and only keep the tiny test for mkisofs, then that's OK for me (and I won't weep :-)